releases/release-1.29/README.md

```diff
@@ -0,0 +1,118 @@
+| Production Readiness Freeze                                   | Enhancements Lead | Thursday 28th September 2023                                                                                          | week 4   | |
+| **Begin [Enhancements Freeze]**                               | Enhancements Lead | [01:00 UTC Friday 6th October 2023 / 18:00 PDT Thursday 5th October 2023](https://everytimezone.com/s/829cf85c)       | week 5   | [master-blocking], [master-informing] |
+| 1.29.0-alpha.3 released                                       | Branch Manager | TBD                                                                                                                      | Week 6   | |
+| Begin Friday APAC-friendly meetings                           | Lead | Friday 20th October 2023                                                                                                           | Week 7   | |
+| 1.29.0-alpha.4 released                                       | Branch Manager | TBD                                                                                                                      | week 8   | [1.29-blocking], [master-blocking], [master-informing] |
+| **Call for [Exceptions][Exception]**                          | Lead | Monday 30th October 2023                                                                                                           | week 9   | |
+| Brace Yourself, Code Freeze is Coming                         | Comms / Bug Triage | Monday 30th October 2023                                                                                             | week 9   | |
+| **Begin Feature blog freeze**                                 | Comms Lead | [01:00 UTC Wednesday 1st November 2023 / 18:00 PDT Tuesday 31st October 2023](https://everytimezone.com/s/24b27b03)          | week 9   | |
+| **KubeCon NA**                                                | | Monday 6th - Thursday 9th November 2023                                                                                                 | week 10  | |
+| **Begin [Burndown]** (Monday, Wednesday, and Friday meetings) | Lead               | Monday 13th November 2023                                                                                            | week 11  | |
+| **Begin [Code Freeze]**                                       | Branch Manager | [01:00 UTC Wednesday 15th November 2023 / 18:00 PDT Tuesday 14th November 2023](https://everytimezone.com/s/b708fa7a)    | week 11  | |
+| 1.29.0-beta.0 released                                        | Branch Manager | Thursday 16th November 2023                                                                                              | week 11  | |
+| Docs deadline — Open placeholder PRs                          | Docs Lead | Thursday 16th November 2023                                                                                                   | week 11  | |
```

Comment:
I'd move this to be at most 7 calendar days after enhancements freeze. As soon as you know your enhancement was accepted, you can push an empty commit and open a WIP PR.

Reply:
Enhancements Freeze is planned for Week 5 (Oct 6)

I've moved `Docs deadline — Open placeholder PRs` to Week 7 (Oct 19) 

These are 2 weeks apart atm -- just so that Docs team have atleast a week time to do some outreach/heads-up about this.